roles/rsyslog/defaults/main.yaml

```diff
@@ -872,15 +879,15 @@ rsyslog__conf_viaq_mmk8s:
                  tls.cacert="{{logging_mmk8s_ca_cert}}"
                  tokenfile="{{logging_mmk8s_token}}" annotation_match=["."])
           }
-        state: '{{ "present"
-               if ("viaq-k8s" in rsyslog__capabilities)
-               else "absent" }}'
 
 rsyslog__conf_viaq_elasticsearch:
 
   - name: 'elasticsearch'
     type: 'output'
     path: '{{rsyslog__viaq_config_dir }}'
+    state: "{{ 'present'
```

Comment:
So ovirt will use the `viaq` capability?
There will be other non-viaq users of the elasticsearch output

Reply:
Shall we add another capability like `elasticsearch` to distinguish from viaq?

Reply:
hmm - actually, this elasticsearch output is specific to viaq `rsyslog__conf_viaq_elasticsearch` - so it should be specific to viaq - we'll have to add another elasticsearch output that is more generic in the future

Reply:
Ok.  I'll leave this output plugin as is for now and open an issue with your comment.  Thanks.